ajax/libs/ui.multiselect/package.json

```
@@ -0,0 +1,35 @@
+{
+  "name": "ui.multiselect",
+  "description": "Another attempt of a sortable, searchable multiple select widget",
+  "filename": "js/ui.multiselect.min.js",
```

```suggestion
  "filename": "js/ui.multiselect.js",
```
This file doesn't appear to exist in the latest tagged version, just the unminified version: https://github.com/michael/multiselect/tree/v0.3.1/js

---

Updated